src/blocks/modal.css

```diff
@@ -191,4 +191,8 @@
     margin: 0;
     padding: 0;
   }
+
```

Comment:
* Please, don't hardcode `height: 322px;` of the `confirmation` modal on 320px https://snipboard.io/CHDASI.jpg . It should not be that high. And the `avatar` modal should not be that high on 320px